app/controllers/collections_controller.rb

```diff
@@ -35,6 +39,8 @@ def create
       )
     )
     Repository.store(updated_cocina_admin_policy)
+    # Close the APO version and reindex
+    version_service.close(description: "Created new collection: #{collection_druid}")
```

Comment:
You should be able to skip the description on close.

Reply:
I wouldn't overwrite an existing version description, but I don't think it really matters much.

Reply:
Fixed to not overwrite an existing version description.